packages/react-dom/src/client/ReactDOM.js

```diff
@@ -739,14 +739,20 @@ const ReactDOM: Object = {
   unstable_flushControlled: DOMRenderer.flushControlled,
 
   __SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED: {
-    // For TapEventPlugin which is popular in open source
-    EventPluginHub,
-    // Used by test-utils
-    EventPluginRegistry,
-    EventPropagators,
-    ReactControlledComponent,
-    ReactDOMComponentTree,
-    ReactDOMEventListener,
+    // Keep in sync with ReactDOMUnstableNativeDependencies.js
+    // and ReactTestUtils.js:
```

Comment:
I think we could use Flow to type the Array below to keep these two files in sync for us, e.g.:

`Events: [Foo, Bar, Baz] = [ ... ]

Reply:
Not sure how many of these methods are typed but we can try.

Reply:
That still wouldn’t help if methods have the same signature but you mess up their order. So I’m not sure it’s that helpful. And if you need to be careful anyway then maybe it’s okay to keep as is.

Reply:
True, it wouldn't be perfect, but even if all the types were the same– it would warn if we added/removed an arg one place and not the other.

Reply:
As discussed, I tried this but some of those are untyped, and it seemed like more work than it's worth